commands/say.go

```diff
@@ -12,7 +13,7 @@ func init() {
 	commands["say"] = sayFromConfig
 }
 
-func sayFromConfig(map[string]any, *sql.DB) (*eelbot.Command, error) {
+func sayFromConfig(map[string]any, *sql.DB, time.Duration) (*eelbot.Command, error) {
```

Comment:
Just curious, but what's the time parameter for? Delay in messages? Do we have to specify it when doing a command, is it optional?

Reply:
The time duration param is for db connection timeouts. Initially we weren't using timeouts, so I added it to better follow best practices.